job_spec/OPERA_DISP_TMS.yml

```diff
@@ -29,25 +17,9 @@ OPERA_DISP_TMS:
           minimum: 1
           maximum: 46986
           example: 11113
-    start_date:
-      api_schema:
-        description: Start of secondary date range for which to generate a tile map (%Y%m%d)
-        type: string
-        pattern: "^\\d{8}$"
-        minLength: 8
-        maxLength: 8
-        example: 20140101
-    end_date:
-      api_schema:
-        description: End of secondary date range for which to generate a tile map (%Y%m%d)
-        type: string
-        pattern: "^\\d{8}$"
-        minLength: 8
-        maxLength: 8
-        example: 20260101
   cost_profiles:
     EDC:
-      cost: 1.0
+      cost: 1000.0
```

Comment:
Is this value based on the biggest job you can run?

Reply:
Yes, 500 frames at ~8 minutes per frame